stdlib/zipfile.pyi

```diff
@@ -274,6 +274,12 @@ if sys.version_info >= (3, 8):
             def joinpath(self, *other: StrPath) -> Path: ...
         else:
             def joinpath(self, add: StrPath) -> Path: ...  # undocumented
+        if sys.version_info >= (3, 12):
+            def glob(self, pattern: str) -> map[Self]: ...
+            def rglob(self, pattern: str) -> map[Self]: ...
+            def is_symlink(self) -> Literal[False]: ...
+            def relative_to(self, other: Self, *extra: StrPath) -> str: ...
```

Comment:
Does `other` _have_ to be an instance of exactly the same class as whatever `self` is? It looks to me like any `zipfile.Path` object will do:

```suggestion
            def relative_to(self, other: Path, *extra: StrPath) -> str: ...
```